File: Python Code/Daily_Flash_Code_0/Week9/16Mar/48-DailyFlash_Solutions/16-Mar/Python/prog5.py
import math
x1 = int(input("x1 : "))
y1 = int(input("y1 : "))
x2 = int(input("x2 : "))
y2 = int(input("y2 : "))
x3 = int(input("x3 : "))
y3 = int(input("y3 : "))
s = math.sqrt((y1 - y2) * (y1 - y2) + (x1 - x2) * (x1 - x2)) + math.sqrt((y3 - y2) * (y3 - y2) + (x3 - x2) * (x3 - x2)) + math.sqrt((y1 - y3) * (y1 - y3) + (x1 - x3) * (x1 - x3)) / 2;
ab = math.sqrt((y1 - y2) * (y1 - y2) + (x1 - x2) * (x1 - x2))
bc = math.sqrt((y3 - y2) * (y3 - y2) + (x3 - x2) * (x3 - x2))
ac = math.sqrt((y1 - y3) * (y1 - y3) + (x1 - x3) * (x1 - x3))
print("R : ", math.sqrt(s * (s - ab)* (s - bc) * (s - ac)) / s)

File: Python Code/Daily_Flash_Code_0/Week9/17Mar/49-DailyFlash_Solutions/17-Mar/Java/prog5.java
import java.util.Scanner;
class Demo {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		float x1, x2, y1, y2, x3, y3;
		double p, a, b, c;
		System.out.println("Enter x1 : ");
		x1 = sc.nextFloat();
		System.out.println("Enter y1 : ");
		y1 = sc.nextFloat();
		System.out.println("Enter x2 : ");
		x2 = sc.nextFloat();
		System.out.println("Enter y2 : ");
		y2 = sc.nextFloat();
		System.out.println("Enter x3 : ");
		x3 = sc.nextFloat();
		System.out.println("Enter y3 : ");
		y3 = sc.nextFloat();
		a = Math.sqrt((y2 - y1) * (y2 - y1) + (x2 - x1) * (x2 - x1));
		b = Math.sqrt((y2 - y3) * (y2 - y3) + (x2 - x3) * (x2 - x3));
		c = Math.sqrt((y3 - y1) * (y3 - y1) + (x3 - x1) * (x3 - x1));
		p = a + b + c;
		System.out.println("X = " + (a * x1 + b * x2 + c * x3) / p + "Y = " + (a * y1 + b * y2 + c * y3) / p);
	}
}
